app/views/quby/v1/paged/_item_question_date.html.haml

```diff
@@ -7,9 +7,6 @@
   .fields{:id => question.html_id}
     - cls = [question.type]
     - cls << "subinput"  if subquestion
-    = text_field :answer, question.day_key, :placeholder => "DD", :size => 2, :autocomplete => question.autocomplete, :class => cls, :disabled => disabled ? "" : nil, data: {field: {key: question.day_key}}
-    = "-"
-    = text_field :answer, question.month_key, :placeholder => "MM", :size => 2, :autocomplete => question.autocomplete, :class => cls, :disabled => disabled ? "" : nil, data: {field: {key: question.month_key}}
-    = "-"
-    = text_field :answer, question.year_key, :placeholder => "JJJJ", :size => 4, :autocomplete => question.autocomplete, :class => cls, :disabled => disabled ? "" : nil, data: {field: {key: question.year_key}}
-
+    - placeholders = Quby::Questionnaires::Entities::Questions::DateQuestion::COMPONENT_PLACEHOLDERS
+    - question.components.each do |component|
+      = text_field :answer, question.send("#{component}_key"), :placeholder => placeholders[component], :size => 4, :autocomplete => question.autocomplete, :class => cls, :disabled => disabled ? "" : nil, data: {field: {key: question.send("#{component}_key")}}
```

Comment:
We laten de streepjes nu weg dus? 
Kunnen we mogelijk oplossen door css als data-field['YYYY']+data-field['MM']:before { content: '-' }